src/binary/level_4_entries.rs

```diff
@@ -131,19 +131,18 @@ impl UsedLevel4Entries {
         }
     }
 
-    /// Returns an unused level 4 entry and marks it as used. If `CONFIG.aslr` is
-    /// enabled, this will return a random available entry.
+    /// Returns the first index of a `num` contiguous unused level 4 entries and marks them as
+    /// used. If `CONFIG.aslr` is enabled, this will return random available entries.
```

Comment:
It might be worth explicitly stating
```suggestion
    /// used. If `CONFIG.aslr` is enabled, this will return random contiguous available entries.
```
for clarity?

Reply:
Agreed, thanks!